CMakeLists.txt

```diff
@@ -170,7 +171,6 @@ if (error EQUAL 0)
   ADD_SUBDIRECTORY(src/lib/xmlParse)
   ADD_SUBDIRECTORY(src/lib/rest)
   ADD_SUBDIRECTORY(src/lib/mongoBackend)
-  ADD_SUBDIRECTORY(src/lib/cache)
```

Comment:
Ok, let's hold for a while. Anyway, even if it is removed it can be checked afterwards (git is a "time machine" :))

NTC